js/index.js

```diff
@@ -1 +1,96 @@
 // Your code goes here
+const btns = document.querySelectorAll(".btn");
+const colorful = document.querySelectorAll("#colorful span");
```

Comment:
good job on the 10 events you picked. There were a few I've never seen/used before so that was cool. 

I'm just going to repeat what I said for DOM I. use more commenting to make your code more easy to read for Human eyes. Your technical ability is super strong but other developers will need to work on your code at some point and managers that hire developers what someone that thinks not just about the code but also about the team. Putting in the time/effort to add white space and comments so it's easy to read and I don't have to spend a lot of effort to hunt down all 10 effects will make you stand out in a job interview.